app/models/search/bookmark_query.rb

```diff
@@ -40,12 +40,49 @@ def exclusion_filters
     @exclusion_filters ||= tag_exclusion_filter
   end
 
-  # Instead of doing a standard query, which would only match bookmark fields
-  # we'll make this a should query that will try to match either the bookmark or its parent
-  def should_query
```

Comment:
https://github.com/otwcode/otwarchive/blob/f274643b0dcca0f4a21f00d23c0f59b2b8096ffd/app/models/search/query.rb#L85-L88

The method `should_query` is still around in the parent class, and used by bookmarkable_query. Should it be removed in those classes?

Reply:
Thanks for taking a look. I was wary of just ripping it out from the parent class and bookmarkable in case they still served a purpose there - though since the one in bookmarkable will never be used because the one in bookmark is gone, and that's the only one remaining, I guess that kind of answers the question 🤔 I'll have to take another look in the morning to make sure I'm reading this right before ripping it out.

Reply:
I've pushed a new version without that code and we'll at least see if all the tests pass (the bookmark specs do locally, at least)